packages/address/src/tests/mocks.test.ts

```diff
@@ -22,7 +22,7 @@ describe('mockCountryRequests', () => {
   });
 
   it('fetches a countries fixture', async () => {
-    const [firstCountry] = await loadCountries('ja');
+    const [firstCountry] = await loadCountries('ja', false);
```

Comment:
As a follow-up, I suggest updating `packages/address-mocks` to handle `includeHiddenZones` in `mockCountryRequests()`, perhaps to return a dummy zone for one `en` country if `includeHiddenZones` is true.

Reply:
When we make the `loadCountries`'s second parameter optional per above, this can be reverted.